tools/internal_ci/linux/presubmit.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+BUILDROOT=${BUILDROOT:-git/transicator}
+export BUILDROOT
+
+# Make a random password for the database we'll be testing with
+TEST_PG_PW=`openssl rand -hex 18`
+export TEST_PG_PW
+
+# Choose a port for PG that probably won't be used.
+pgPort=54321
+
+# Make a temporary GOPATH to build in
+gobase=`mktemp -d`
+base=${gobase}/src/github.com/apigee-labs/transicator
+GOPATH=${gobase}
+export GOPATH
+
+base=${GOPATH}/src/github.com/apigee-labs/transicator
+mkdir -p ${base}
+(cd ${BUILDROOT}; tar cf - .) | (cd ${base}; tar xf -)
+
+finalResult=0
+
+(cd ${base}; make presubmit)
+finalResult=$?
+
+if [ $finalResult -eq 0 ]
+then
+  dbName=transicator-pg-$$
+  # Build and launch postgresql image
+  (cd ${base}; docker build --tag ${dbName} ./pgoutput)
+  docker run -d \
+    --name ${dbName} \
+    -e POSTGRES_PASSWORD=${TEST_PG_PW} \
+    -p ${pgPort}:5432 \
+    ${dbName}
+
+  # Give PG a few seconds to start
+  sleep 5
```

Comment:
5 seconds seems good, although I have seen it sometimes take a little longer...(not sure why).